File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/operation/meta/schema/read/AttReadException.java
package ru.citeck.ecos.records3.record.operation.meta.schema.read;

public class AttReadException extends RuntimeException {

    public AttReadException(String alias, String attribute, String message) {
        super("Attribute read error: " + message + ". Alias: '" + alias + "' Attribute: '" + attribute + "'");
    }
}

File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/atts/schema/read/AttReadException.kt
package ru.citeck.ecos.records3.record.atts.schema.read

import ru.citeck.ecos.records3.record.atts.schema.exception.AttSchemaException

class AttReadException(alias: String, attribute: String, message: String) :
    AttSchemaException("Attribute read error: $message. Alias: '$alias' Attribute: '$attribute'")
